TransformationSystem/Client/TaskManager.py

```diff
@@ -333,6 +335,116 @@ def __init__( self, transClient = None, logger = None, submissionClient = None,
     self.destinationPlugin_o = None
 
   def prepareTransformationTasks( self, transBody, taskDict, owner = '', ownerGroup = '', ownerDN = '' ):
+
+    bulkFlag = self.opsH.getValue( "Transformations/BulkSubmission", False )
```

Comment:
This flag should be per transformation type, otherwise it ends up applying for every transformation, including reconstruction and others. This is not what we want. To be honest for today I would put it as a parameter of the WorkflowTaskAgent, and simply passed in here as
```
def prepareTransformationTasks( self, transBody, taskDict, owner = '', ownerGroup = '', ownerDN = '', bulkFlag = False ):
```